sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2145,11 +2145,17 @@ ALTER TABLE `patient_data` ADD COLUMN `care_team_facility` text;
 #EndIf
 
 #IfRow2D layout_options form_id DEM field_id care_team
-UPDATE `layout_options` SET field_id='care_team_provider', seq=22, group_id=3, title='Care Team (Provider)', data_type=45 WHERE form_id='DEM' AND field_id='care_team';
+SET @group_name = (SELECT group_name FROM layout_options WHERE field_id='care_team' AND form_id='DEM');
+SET @backup_group_name = (SELECT group_name FROM layout_options WHERE field_id='DOB' AND form_id='DEM');
+SET @seq = (SELECT MAX(seq) FROM layout_options WHERE group_name = IFNULL(@group_name,@backup_group_name) AND form_id='DEM');
+UPDATE `layout_options` SET field_id='care_team_provider', group_name=IFNULL(@group_name,@backup_group_name), seq=@seq+1, title='Care Team (Provider)', data_type=45 WHERE form_id='DEM' AND field_id='care_team';
 #EndIf
 
 #IfNotRow2D layout_options form_id DEM field_id care_team_facility
-INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`,`fld_rows`) VALUES ('DEM', 'care_team_facility', '3', 'Care Team (Facility)', 22, 44, 1, 0, 0, '', 1, 1, '', '', '', 0);
+SET @group_name = (SELECT group_name FROM layout_options WHERE field_id='care_team_provider' AND form_id='DEM');
+SET @backup_group_name = (SELECT group_name FROM layout_options WHERE field_id='DOB' AND form_id='DEM');
+SET @seq = (SELECT MAX(seq) FROM layout_options WHERE group_name = IFNULL(@group_name,@backup_group_name) AND form_id='DEM');
+INSERT INTO `layout_options` (`form_id`,`field_id`,`group_id`,`title`,`seq`,`data_type`,`uor`,`fld_length`,`max_length`,`list_id`,`titlecols`,`datacols`,`default_value`,`edit_options`,`description`,`fld_rows`) VALUES ('DEM', 'care_team_facility', IFNULL(@group_name,@backup_group_name), 'Care Team (Facility)', @seq+1, 44, 1, 0, 0, '', 1, 1, '', '', '', 0);
```

Comment:
TODO for me:
I think i may of given you some outdated logic (group_name instead of group_id in places). If i did, I'll fix that when I test it and bring it in.